pkger/testdata/dashboard_single_stat_plus_line.json

```diff
@@ -69,4 +70,4 @@
       ]
     }
   }
-]
+]
```

Comment:
oh no, my newline 😢

Reply:
Ack, my IDE obeys the `.editorconfig` and the InfluxDB is rather lacking; so they're purged :'(